tools/downloader/requirements-pytorch.in

```diff
@@ -7,6 +7,6 @@ scipy           # via torchvision
 # which might undo the effects of the first change. It's not yet been included in any
 # release at the time of writing, so whether it helps, remains to be seen.
 # For now, just cap the PyTorch version at 1.6.x.
-torch>=1.5,!=1.7.*
+torch>=1.8
```

Comment:
The comment above is now obsolete, so please remove it.

Reply:
Done